components/solutions/featureBanner.tsx

```diff
@@ -9,7 +9,7 @@ export interface FeatureArray extends Array<Feature> {}
 
 export default function FeatureBanner({
   features,
-  category = "Customer Data Platform",
+  category = "Data Tool",
```

Comment:
Used for multiple comparison pages (Segment, Census, High Touch) so didn't refer to reverse ETL here either.

Reply:
Same comment as above. "Reverse ETL" works just fine in this case imo.